.github/workflows/deploy.yml

```diff
@@ -29,11 +29,9 @@ jobs:
           python-version: 3.x
       - name: Install dependencies
         run: |
-          set -xeu
-          python -VV
-          pip install setuptools wheel
+          pip install build
```

Comment:
You can write that in a single line : `run: pip install build`

Reply:
true